libs/sdk-ui/src/base/localization/bundles/de-DE.json

```diff
@@ -104,19 +104,29 @@
     "filters.nextNQuarters": "{n, plural, one {Nächstes # Quartal} other {Nächste # Quartale}}",
     "filters.nextNYears": "{n, plural, one {Nächstes # Jahr} other {Nächste # Jahre}}",
     "filters.interval.days.past": "Von {from} bis {to} {to, plural, one {Tag} other {Tage}} vorher",
+    "filters.interval.days.past.sameValue": "vor {value} {value, plural, one {Tag} other {Tagen}}",
```

Comment:
`vor` (lower case) vs `Von`
@patrikbraborec , please make the first letter uppercase in the PR - applicable also for the strings below

Reply:
We'll report the bug after